srcpkgs/brother-hl5340d-cupswrapper/files/LICENSE

```diff
@@ -0,0 +1,9 @@
+License Agreement
```

Comment:
Licenses should not be vendored. Is it included in the deb package? vlicense it from there.